lib/portlayer/exec/container_cache.go

```diff
@@ -48,11 +61,35 @@ func NewContainerCache() {
 func (conCache *containerCache) Container(idOrRef string) *Container {
 	conCache.m.RLock()
 	defer conCache.m.RUnlock()
-	// find by id
+	// find by id or moref
 	container := conCache.cache[idOrRef]
 	return container
 }
 
+// return an array of pointers to the containers in the cache
+// TODO: implement filtering here?
+func (conCache *containerCache) Containers(allStates *bool) []*Container {
```

Comment:
Not sure why we need this in cache pkg. AFAIS there is only one caller so this code very well could live in the caller. Also please bool not *bool.